oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json

```diff
@@ -214,7 +214,7 @@
                   "i": "15",
                   "type": "Widget",
                   "expressions": [
-                    "meter_pulsar_storage_read_rate.sum(['cluster','node'])"
+                    "meter_pulsar_storage_read_rate"
```

Comment:
Doesn't `pulsar-cluster` require `aggregate_labels ` to aggregate the nodes?

Reply:
I'd like to know more about it, for the overview of clusters, is the method we usually recommend is aggregation display?

Reply:
Yes, especially the service could have many instances.  If you want to show the all instances overall info, you should
put them on the `Instance List` widget.

Reply:
Learned, thanks.